runtime/gc_base/RootScanner.hpp

```diff
@@ -91,6 +91,9 @@ class MM_RootScanner : public MM_BaseVirtual
 #endif /* J9VM_GC_MODRON_SCAVENGER */	 	
 	bool _classDataAsRoots; /**< Should all classes (and class loaders) be treated as roots. Default true, should set to false when class unloading */
 	bool _includeJVMTIObjectTagTables; /**< Should the iterator include the JVMTIObjectTagTables. Default true, should set to false when doing JVMTI object walks */
+#if defined(J9VM_GC_ENABLE_DOUBLE_MAP)
+	bool _includeDoubleMap; /**< Should the GC policy be balanced and compressed refs (explicitly enabled via command line parameter). Default is false. */
+#endif /* J9VM_GC_ENABLE_DOUBLE_MAP */
```

Comment:
This does not make sense: "Should the GC policy be balanced and compressed refs"!?

The other part about command line and default value should not be even mentioned.